news/2023-01-01-osumonthly-6.md

```diff
@@ -0,0 +1,331 @@
+
+Next up we have Motion's 4 miss on the highest difficulty of [XHRONOXAPSULE](https://osu.ppy.sh/scores/fruits/210576459), a map that currently sits as the ranked map with the highest star rating in the game, coming in at 10.07 stars. The scariest and toughest part of this map is definitely the end section which features 277bpm antiflow patterns, what makes this play so "special" is that he actually hit the hardest part of the map, a section that no other player had hit before, but failed at a previous, easier section, turning this play into a reverse choke. However, this isn't the play that sits in the leaderboard, since shortly after Motion would set yet another high score with better combo but 3 more misses, all located at the end of the map.
+
+The [Shin dan](https://osu.ppy.sh/beatmapsets/884597#fruits/1872971) might just be the hardest map compilation in osu!catch history. After all, surviving through a 10.43 star map that lasts for almost 10 minutes doesn't seem like an easy feat, however, after a great community effort, [ArMa79](https://osu.ppy.sh/users/4982799/fruits) claimed the first clear on this map, which in this context means, obtaining a 99% on the map. This isn't the end of the story though, as only a couple months later, Motion too would rise up to the challenge, sniping ArMa79's score in the process and setting one of the best plays of 2022.
+
+[ExGon](https://osu.ppy.sh/users/214187)'s score on [Ded_ed](https://osu.ppy.sh/beatmapsets/1705910#fruits/3800941) is probably the most unusual and shocking play set this year, but what makes this play so interesting? From an outside perspective, this map may just look like an easy 33 second, 3 star map, but this couldn't be further from the truth. This map features an astoundingly high CS 7, combined with strange pixel patterns that requires immense precision that only become harder after applying HR. If this already sounds hard, wait until you hear that he did all of this while using flashlight, absolute madness!
+
+When a player as good as [YesMyDarknesss](https://osu.ppy.sh/users/4158549) takes over 2000 attempts to fc a single map, you know they mean business. This play on [Palace of Melancholia](https://osu.ppy.sh/beatmapsets/1596787#fruits/3261247) isn't just one of the best of this year, but one of the best HR scores ever performed, every single section of this map is absolutely brutal, requiring immense catcher control and consistency. However, this is not the end of the story for YesMyDarknesss, as to this very day he's still trying to get another FC on this map, but this time with HDHR, having invested yet another 1000 attempts on this quest, we wish him the best for this grind in 2023!
```

Comment:
no link to the play?

Reply:
https://github.com/ppy/osu-wiki/pull/8570/commits/51385943ddb319ef202a6c8ca03aa2b2ff6467a2